devicetypes/heltun/heltun-ht01-thermostat.src/heltun-ht01-thermostat.groovy

```diff
@@ -0,0 +1,538 @@
+	cmds << zwave.thermostatModeV2.thermostatModeSupportedGet().format() //get supported modes
+	sendHubCommand(cmds, 1200)
+	runIn(15, "checkParam")
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.multichannelassociationv2.MultiChannelAssociationReport cmd) {
+	def cmds = []
+	if (cmd.groupingIdentifier == 1) {
+		if (cmd.nodeId != [1]) {
+			cmds << zwave.multiChannelAssociationV2.multiChannelAssociationRemove(groupingIdentifier: 1).format()
+			cmds << zwave.multiChannelAssociationV2.multiChannelAssociationSet(groupingIdentifier: 1, nodeId: 1).format()
+		}
+	}
+	if (cmds) sendHubCommand(cmds, 1200)
+}
+
+def configure() {
+	ping()
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.clockv1.ClockReport cmd) {
+	def currDate = Calendar.getInstance(location.timeZone)
+	def time = [hour: currDate.get(Calendar.HOUR_OF_DAY), minute: currDate.get(Calendar.MINUTE), weekday: currDate.get(Calendar.DAY_OF_WEEK)]
+	if ((time.hour != cmd.hour) || (time.minute != cmd.minute) || (time.weekday != cmd.weekday)){
+		sendHubCommand(zwave.clockV1.clockSet(time).format())
+	}
+}
+
+def resetEnergyMeter() {
+	sendHubCommand(zwave.meterV3.meterReset().format())
+}
+
+def off() {
+	setThermostatMode("off")
+}
+
+def heat() {
+	setThermostatMode("heat")
+}
+
+def emergencyHeat() {
+	setThermostatMode("emergencyHeat")
+}
+
+private parameterMap() {[
+
+[title: "Display Brightness Control", description: "The HE-HT01 can adjust its display brightness automatically depending on the illumination of the ambient environment and also allows to control it manually.",
+ name: "Selected Brightness Level", options: [
+			0: "Auto",
+			1: "Level 1 (Lowest)",
+			2: "Level 2",
+			3: "Level 3",
+			4: "Level 4",
+			5: "Level 5",
+			6: "Level 6",
+			7: "Level 7",
+			8: "Level 8",
+			9: "Level 9",
+			10: "Level 10 (Highest)"
+    ], paramNum: 5, size: 1, default: "0", type: "enum"], 
+    
+[title: "Touch Sensor Sensitivity Threshold", description: "This Parameter allows to adjust the Touch Buttons Sensitivity. Note: Setting the sensitivity too high can lead to false touch detection. We recommend not changing this Parameter unless there is a special need to do so.",
+ name: "Selected Touch Sensitivity", options: [
+			1: "Level 1 (Low sensitivity)",
+			2: "Level 2",
+			3: "Level 3",
+			4: "Level 4",
+			5: "Level 5",
+			6: "Level 6",
+			7: "Level 7",
+			8: "Level 8",
+			9: "Level 9",
+			10: "Level 10 (High sensitivity)"
+    ], paramNum: 6, size: 1, default: "6", type: "enum"], 
+    
+[title: "Relay Output Mode", description: "This Parameter determines the type of load connected to the device relay output. The output type can be NO – normal open (no contact/voltage switch the load OFF) or NC - normal close (output is contacted / there is a voltage to switch the load OFF)",
+ name: "Selected Mode", options: [0: "NO - Normal Open", 1: "NC - Normal Close"], paramNum: 7, size: 1, default: "0", type: "enum"],
```

Comment:
Could You reformat it so `options` would look the same way as in other parameters ?